src/utils/logger.js

```diff
@@ -28,6 +28,12 @@ const logger = createLogger({
 logger.add(new transports.File({ filename: errLog, level: 'error' }));
```

Comment:
winston.transports.Console() for dev only or below?

Reply:
we are currently doing that

```
if (process.env.NODE_ENV !== 'production') {
  logger.add(new transports.Console({
    level: 'debug',
  }));
}
```